chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/api/RoomApiType.kt

```diff
@@ -1,9 +1,9 @@
-package com.pusher.chatkit.rooms
+package com.pusher.chatkit.rooms.api
 
 import com.google.gson.annotations.SerializedName
 import com.pusher.chatkit.CustomData
 
-data class Room(
+internal data class RoomApiType(
```

Comment:
we already have ApiTypes for that: JoinedRoomApiType and NotJoinedRoomApiType which are indeed immutable data classes with less of optionality

Reply:
indeed this type here is only used in UserInternalEvent which should be gone

Reply:
So to clarify - should I remove the RoomApiType and refactor the UserInternalEvent?

Reply:
UserInternalEvent should be removed, likewise this file here

Reply:
👍 have removed both!